srcpkgs/unrar/template

```diff
@@ -12,13 +12,12 @@ checksum=505c13f9e4c54c01546f2e29b2fcc2d7fabc856a060b81e5cdfe6012a9198326
 repository=nonfree
 
 do_build() {
-	vsed -e 's/^\(CXXFLAGS\)=\(.*\)/\1+=\2/' \
+	vsed -e 's/^\(CXXFLAGS\)=-march=native \(.*\)/\1+=\2/' \
 		 -e 's/^\(LDFLAGS\)=\(.*\)/\1+=-lpthread \2/' \
 		 -i makefile
 	make CXX="$CXX" LD="$LD" STRIP=: -f makefile
-	# early install of unrar because makefile deletes unrar on lib creation
+	# early install of unrar because need to clean for lib creation
```

Comment:
This does not look related to patch subject

Reply:
Its just a small comment correction (along with the other comment change), but if thats not appropriate to include in the PR then I'll remove it.

Reply:
I'm no authority on the subject, that was just a comment while passing by, feel free to ignore